infrastructure/s3_event_bridge_lambda.tf

```diff
@@ -0,0 +1,203 @@
+#####################################################
+# Variables
+#####################################################
+variable "DEPLOY_VECTOR_AUTOMATION" {
+  type    = bool
+  default = false
+}
```

Comment:
do we need this if it's defined in the variables file? also, it's uppercase 😯 I think the idiomatic style is lowercase

Reply:
good catch! We don't need this since it's already in `variables.tf`. I'll delete it